src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Reload/ReloadableProject.cs

```diff
@@ -143,7 +153,13 @@ public Task Initialize()
                         // The only safe thing to do at this point is to reload the project in the solution
                         // TODO: should we have an additional return value here to indicate that the existing project could be in a bad
                         // state and the reload needs to happen without the user being able to block it?
-                        _telemetryService.Report("projectreload/reload-failed", "Project Reload Failure due to general fault", ex);
+                        var correlationEvent = _telemetryService.Report(ReloadFailedNFWPath, "Project Reload Failure due to general fault", ex);
+                        _telemetryService.PostOperation(
```

Comment:
you don't need to. you can put PostOperation on every exit point with either TelemetryResult.Failed or succeed.

either using or try/finally make those easier though since compiler make sure it gets called for every exit point.

but that's just coding pattern, choose one you prefer. but I see no event for succeeded case. is that fired in other place?